Bottom: 8c6f5a3578b373d6eef91bb9cd0bd12fb25578b6
Top:    8abb77054cc6a74ee11812cd4f2b16f6bcc779a2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 15:21:05 +0000

Adjust STEP export to 2.3mm board thickness (Hack - need something quick!)


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index c736af7..5dafc05 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -31,7 +31,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 
 static int step_set_layer (const char *name, int group, int empty);
 static void use_gc (hidGC gc);
